x-pack/plugins/features/common/sub_feature.ts

```diff
@@ -29,6 +29,11 @@ export interface SubFeatureConfig {
 
   /** Collection of privilege groups */
   privilegeGroups: readonly SubFeaturePrivilegeGroupConfig[];
+
+  /**
```

Comment:
__nit:__ it'd make sense to unify JSDocs with the one that we have for `description` in `kibana_feature` (these should be basically the same except `feature` -> `sub-feature`).